validate-deployment-readiness

```diff
@@ -117,9 +109,9 @@ async function main() {
   }
   console.log(`${yellow}Deployed hash: ${reset}${deployedHash}`);
 
-  const latestLocalMigration = await getLocalMigration(migrationPath);
-  const localHash = hash.update(latestLocalMigration).digest('hex');
-  console.log(`${yellow}Local hash:    ${reset}${localHash} (from file: ${latestLocalMigration})`);
+  const localList = await getLocalList(migrationPath);
+  const localHash = hash.update(localList.join('\n')).digest('hex');
```

Comment:
I assume we're adding the line breaks because the server will have line breaks?

Reply:
Yep — https://github.com/ca-la/api/pull/309